src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/FilterVariantTranches.java

```diff
@@ -133,8 +139,8 @@ public void firstPassApply(final VariantContext variant, final ReadsContext read
 
     @Override
     public void afterFirstPass() {
-        logger.info(String.format("Found %d SNPs %d indels with INFO score key:%s.", scoredSnps, scoredIndels, infoKey));
-        logger.info(String.format("Found %d SNPs %d indels in the resources.", snpScores.size(), indelScores.size()));
+        logger.info(String.format("Found %d SNPs and %d indels with INFO score key:%s.", scoredSnps, scoredIndels, infoKey));
+        logger.info(String.format("Found %d SNPs and %d indels in the resources.", snpScores.size(), indelScores.size()));
```

Comment:
For clarity can you rename snpScores/indelScores to something like resourceSNPScores/resourceIndelScores or truthSNPScores/truthIndelScores.

Reply:
fixed